app/_src/kubernetes-ingress-controller/guides/configuring-health-checks.md

```diff
@@ -2,355 +2,301 @@
-when passive health-checks marked it unhealthy.
+1.  Update the KongIngress resource to use active health-checks. 
+  ```bash
+    echo "apiVersion: configuration.konghq.com/v1
+    kind: KongIngress
+    metadata:
+        name: demo-health-checking
+    upstream:
+      healthchecks:
+        active:
+          healthy:
+            interval: 5
+            successes: 3
+          http_path: /status/200
+          type: http
+          unhealthy:
+            http_failures: 1
+            interval: 5
+        passive:
+          healthy:
+            successes: 3
+          unhealthy:
+            http_failures: 3" | kubectl apply -f -
+    ```
+    The results should look like this:
+    ```text
+    kongingress.configuration.konghq.com/demo-health-checking created
+    ```      
+    
+    This configures Kong to actively probe `/status/200` every 5 seconds. If a Pod is unhealthy from Kong's perspective, 3 successful probes changes the status of the Pod to healthy and Kong again starts to forward requests to that Pod.
+
+1. Test the Ingress rule.
+    ```bash
+    $ curl -i $PROXY_IP/test/status/200
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 200 OK
+    Content-Type: text/html; charset=utf-8
+    Content-Length: 0
+    Connection: keep-alive
+    Server: gunicorn/19.9.0
+    Access-Control-Allow-Origin: *
+    Access-Control-Allow-Credentials: true
+    X-Kong-Upstream-Latency: 2
+    X-Kong-Proxy-Latency: 0
+    Via: kong/3.1.1
+    ```            
+1.  Trip the circuit again by sending three requests that returns the status 500 from httpbin.
+    ```bash
+    $ curl -i $PROXY_IP/test/status/500
+    $ curl -i $PROXY_IP/test/status/500
+    $ curl -i $PROXY_IP/test/status/500
+    ```
+
+    When you send the requests, it fails for about 15 seconds, the duration for active health checks to re-classify the httpbin Pod as healthy again.
+
+    ```bash
+    $ curl -i $PROXY_IP/test/status/200
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 503 Service Temporarily Unavailable
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 62
+    X-Kong-Response-Latency: 1
+    Server: kong/3.3.1
+    
+    {
+      "message":"failure to get a peer from the ring-balancer"
+    }%                  
+    ```          
+1.  Send the requests after 15 seconds or so.
+
+    ```bash
+    $ curl -i $PROXY_IP/test/status/200
+    ```
+    The results should look like this:
+    ```text
+    HTTP/1.1 200 OK
+    Content-Type: text/html; charset=utf-8
+    Content-Length: 0
+    Connection: keep-alive
+    Server: gunicorn/19.9.0
+    Access-Control-Allow-Origin: *
+    Access-Control-Allow-Credentials: true
+    X-Kong-Upstream-Latency: 2
+    X-Kong-Proxy-Latency: 0
+    Via: kong/3.1.1
+    ```            
+
+Active health-checks automatically marked a Pod as healthy when passive health-checks marked it unhealthy.
 
 ## Bonus
```

Comment:
I think we should tell the reader what the behaviour is here (each proxy pod keeps track of it's own healthchecks), or remove the bonus section.